python/smqtk/utils/factors.py

```diff
@@ -6,62 +6,76 @@
 __all__ = ["sieve_of_eratosthenes", "prime_factors", "factors", "factor_pairs"]
 
 
+# Set of known prime values
+_soe_prime_cache = [2, 3]
```

Comment:
Leaving the 2 and 3 in the base cache that is added to instead of adding a list addition operation upon every return. The effect is the same either way and I think not doing list operations is more efficient (reduces overall list copies).